doc/EnvironmentalMonitoring/Monitoring-architecture.md

```diff
@@ -102,12 +103,13 @@ The following sensors produce monitoring data:
   - data is refreshed once per minute
 - `CertificateInfoSensor`
   - data: information about certificates associated with this security server
-    - id
-    - issuer DN
-    - subject DN
+    - certificate SHA-1 hash
     - validity period: not before (ISO 8601 date)
     - validity period: not after (ISO 8601 date)
-    - status
+    - the type of the certificate:
+       - `AUTH_OR_SIGN` for the Security Server Owner (member) certificate (sign) and the Security Server certificate (auth)
```

Comment:
Why "Security Server Owner"? Doesn't this list also non-owner member sign certificates?